pages/linux/brew.md

```diff
@@ -14,14 +14,18 @@
 
 `brew list`
 
-- Update an installed formula (passing no formula updates all installed formulae):
+- Update an installed formula (passing no `{{formula}}` updates all installed formulae):
 
 `brew upgrade {{formula}}`
 
-- Update brew itself:
+- Fetch the newest version of Linuxbrew and all formulae from GitHub:
 
 `brew update`
 
+- Show formulae that have an updated version available:
```

Comment:
Hrm. I only suggested it because the term I've usually seen around is `new version`, rather than `updated version`. While it is true that `new` can signify a new package / thing, if qualified by `version` it only refers to existing entities.